docs/source/dataframe-groupby.rst

```diff
@@ -220,3 +220,18 @@ Pandas' ``groupby('g').agg(list)``:
    ...     agg=lambda s0: s0.apply(lambda chunks: list(it.chain.from_iterable(chunks))),
    ... )
    >>> df.groupby('g').agg(collect_list)
+
+To apply :py:class:`dask.dataframe.groupby.SeriesGroupBy.nunique` to more than one
+column you can use:
+
+.. code-block:: python
+
+    >>> nunique = dd.Aggregation(
+    ...     name="nunique",
+    ...     chunk=lambda s : s.apply(lambda x: list(set(x))),
+    ...     agg=lambda s0 : s0._selected_obj.groupby(
+    ...         level=list(range(s0._selected_obj.index.nlevels))
+    ...     ).sum(),
```

Comment:
Python doesn't support multi-line lambdas, so this is a syntax error. Perhaps define a new function, or drop the line-wrap?

Reply:
It also feels a bit odd to use a hidden attribute (`_selected_obj`) here. cc @TomAugspurger for thoughts.

Reply:
I think we should be able to do this with `obj` instead of `_selected_obj`